integration-tests/src/test/java/oracle/weblogic/kubernetes/ItClusterResourceScaling.java

```diff
@@ -0,0 +1,527 @@
+import static oracle.weblogic.kubernetes.utils.PodUtils.checkPodExists;
+import static oracle.weblogic.kubernetes.utils.PodUtils.checkPodReady;
+import static oracle.weblogic.kubernetes.utils.PodUtils.getExternalServicePodName;
+import static oracle.weblogic.kubernetes.utils.PodUtils.setPodAntiAffinity;
+import static oracle.weblogic.kubernetes.utils.SecretUtils.createSecretWithUsernamePassword;
+import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Simple JUnit test file used for testing operator usability.
+ * Use Helm chart to install operator(s)
+ */
+@DisplayName("Test scaling usability ")
+@IntegrationTest
+@Tag("olcne")
+@Tag("oke-parallel")
+@Tag("kind-parallel")
+@Tag("okd-wls-mrg")
+class ItClusterResourceScaling {
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+
+  // domain constants
+  private final String domainUid = "usabdomain";
+
+  private final String clusterName = "cluster-1";
+  private final int replicaCount = 2;
+  private final String adminServerPrefix = "-" + ADMIN_SERVER_NAME_BASE;
+  private final String managedServerPrefix = "-" + MANAGED_SERVER_NAME_BASE;
+  private boolean isDomainRunning = false;
+  private String adminSvcExtRouteHost = null;
+
+  private static LoggingFacade logger = null;
+
+  /**
+   * Get namespaces for operator, domain.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // get a unique domain namespace
+    logger.info("Getting a unique namespace for WebLogic domain 2");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+    createTestRepoSecret(domainNamespace);
+  }
+
+  /**
+   * Install the Operator successfully.
+   * Create domain and verify the domain is started
+   * Verify cluster can be scale via Rest Api
+   * Verify expected http responce in case of :
+   * 1. Provided invalid auth token
+   * 2. Provided invalid domainuid
+   * 3. Provided invalid clustername
+   * 4. Provided above max replica
+   * 5. Missed header
+   * 6 Missed auth header
+   */
+  @Test
+  @DisplayName("Create domain, managed by operator and domain1, perfom scaling operation via REST  "
+      + "verify scaling operation generate expected code in case of negative scenarios: bad auth"
```

Comment:
add space at the end of line:
"verify scaling operation generate expected code in case of negative scenarios: bad auth" ->
"verify scaling operation generate expected code in case of negative scenarios: bad auth "

Reply:
done